chain/cosmos/cosmos_chain.go

```diff
@@ -345,6 +345,11 @@ func (c *CosmosChain) SendFunds(ctx context.Context, keyName string, amount ibc.
 	return c.getFullNode().BankSend(ctx, keyName, amount)
 }
 
+// Implements Chain interface
+func (c *CosmosChain) SendFundsWithNote(ctx context.Context, keyName string, amount ibc.WalletAmount, note string) (string, error) {
+	return c.getFullNode().BankSendWithNote(ctx, keyName, amount, note)
+}
```

Comment:
Should we just modify BankSend with an `extraFlags ...string`?

I am good with adding this, but wonder if we really need 2 bank sends with just a note memo variant. UX wise, I can just mention in the release notes for next ICT v8.

Reply:
I talked with @agouin about this and I, at least, had a preference to not break the SendFunds API for everyone else. But if we should bite the bullet and just break it now, I'm okay with that too. There just could be many calls to it in test cases out there.

Reply:
If we add `extraFlags ...string` it's not function signature breaking :) since its just optional

I think we add this & can reduce. It should only break our internal implementations which need updating

User facing functions wont change or require migration work

Reply:
I can take this on in another PR if you prefer

Reply:
I like that, but also forgot to include that we need to return the tx hash too... If the note/memo results in external processes, that tx hash is used to check the state. Again, something SendFunds() should probably do, but that will break the API.

Reply:
gotcha, Ill make a ticket then and can decide in a future release. Or we wait to do that in the v52 refactor